tools/dockerfile/grpc_artifact_python_musllinux_1_1_x64/Dockerfile

```diff
@@ -35,3 +36,5 @@ RUN curl -sSL -o ccache.tar.gz https://github.com/ccache/ccache/releases/downloa
     && make -j4 && make install \
     && cd ../.. \
     && rm -rf ccache-4.5.1 ccache.tar.gz
+
+RUN git config --global --add safe.directory /var/local/jenkins/grpc
```

Comment:
How do we know it's `/var/local/jenkins/grpc`? Is this $HOME? Should we then use the env var?

Reply:
It's not home. As far as I know, there is no env var provided for this. It's simply hardcoded in the docker images. We could set an env var in `run_tests.py`, but I think that refactor is out of the scope of this PR.